nav2_sms_behavior/CMakeLists.txt

```diff
@@ -19,8 +19,11 @@ find_package(builtin_interfaces REQUIRED)
 find_package(tf2_ros REQUIRED)
 find_package(nav2_costmap_2d REQUIRED)
 find_package(nav2_core REQUIRED)
-find_package(nav2_recoveries REQUIRED)
+find_package(nav2_behaviors REQUIRED)
 find_package(pluginlib REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
+find_package(action_msgs REQUIRED)
+find_package(CURL REQUIRED)
```

Comment:
Why CURL?

Reply:
use curl in  Twillio API(twillio.cpp)
if i don't use package curl , failed to send sms.

Reply:
`CURL`
->
`curl`

Reply:
how can i change package CURL?